packages/markerclustererplus/src/markerclusterer.ts

```diff
@@ -1159,4 +1161,24 @@ export class MarkerClusterer extends OverlayViewSafe {
       title: ""
     };
   }
+
+  /**
+   * Generates default styles augmented with user passed values.
+   * Useful when you want to override some default values but keep untouched
+   *
+   * @param overrides override default values
+   */
+  static creteDefaultStyle(overrides: ClusterIconStyle): ClusterIconStyle {
```

Comment:
```suggestion
  static createDefaultStyle(overrides: ClusterIconStyle): ClusterIconStyle {
```

Reply:
Good catch, I'm thinking maybe rename this method to `withDefaultStyle`, sounds closer to the what method is doing

Reply:
Still need to fix this with either of the two, correct spelling or `withDefaultStyle`.